modules/ossm-rn-known-issues.adoc

```diff
@@ -62,6 +62,10 @@ $ oc -n istio-system get service istio-ingressgateway -o jsonpath='{.status.load
 
 * link:https://issues.jboss.org/browse/MAISTRA-158[MAISTRA-158] Applying multiple gateways referencing the same hostname will cause all gateways to stop functioning.
 
+* link:https://issues.jboss.org/browse/MAISTRA-806[MAISTRA-806] Evicted Istio Operator Pod causes Mesh and CNI not to deploy.
```

Comment:
I'm not familiar with Mesh usage. Should we spell out Service Mesh or is there a replaceable for the name like {product-name} or similar?

Reply:
I believe this is referring to a lowercase service mesh rather than the product name. But it's also the exact title of the known issue in jboss, so I copied it as is because a lot of the other links on this page list the link and the title of the issue.

Would making that mesh lowercase work?

Reply:
I think that because this line is referencing an issue, it's fine the way it is.

Reply:
If you don't think it's referencing the product name itself, then lowercase mesh would work here. Thanks!

Reply:
Ok, mesh is now lowercase and the commits are squashed.